frontend/src/citizen-frontend/messages/state.tsx

```diff
@@ -67,21 +65,18 @@ export const isRegularThread = (
 ): thread is CitizenMessageThread.Regular => thread.type === 'Regular'
 
 const markMessagesReadByThreadId = (
-  threads: CitizenMessageThread[],
+  thread: CitizenMessageThread,
   threadId: UUID
-): CitizenMessageThread[] =>
-  threads.filter(isRegularThread).map(
```

Comment:
There's a bug here that removes all the redacted threads from react-query cache when a regular thread is marked as read. It's fixed by this PR.